nut/DOCS.md

```diff
@@ -35,7 +35,8 @@ comparison to installing any other Home Assistant add-on.
 1. Configure the `users` and `devices` options.
 1. Start the "Network UPS Tools" add-on.
 1. Check the logs of the "Network UPS Tools" add-on to see if everything went well.
-1. Configure the [NUT Integration][nut-ha-docs].
+1. Configure the [NUT Integration][nut-ha-docs] using Host `a0d7b954-nut`,
+   Port `3493`, and the Username/Password configured in the add-on.
```

Comment:
As I've commented in the previous PR as well, the hostname isn't a fixed thing. It can differ depending how you install it.

These instructions are this not correct.

../Frenck

Reply:
Yeah, this hostname was already in the documentation in a few places, so I needed to figure out how to fix the existing docs before I could fix my new docs.

I think I've figured it out.  Does this new commit look better?